lynx/common/actions/print.py

```diff
@@ -10,6 +10,7 @@ class Print(Action):
     """
     text: str = ""
     user_id: str = ""
+    object_id: int = -1

```

Comment:
Co to za zmiana?

Reply:
Chciałem, żeby podczas printowania czegoś przez konkretnego agentaa wyświelać jego id
![image](https://github.com/group-project-gut/lynx-python-common/assets/101421631/c5ed7ccc-f052-44cb-9839-1e95f709e779)

Reply:
Ok, fair